src/benchmark/java/com/commercetools/sync/benchmark/InventorySyncBenchmark.java

```diff
@@ -57,25 +55,23 @@ public static void tearDown() {
 
     @Test
     public void sync_NewInventories_ShouldCreateInventories() throws IOException {
+        // preparation
         final List<InventoryEntryDraft> inventoryEntryDrafts = buildInventoryDrafts(NUMBER_OF_RESOURCE_UNDER_TEST);
-
-
         final InventorySyncOptions inventorySyncOptions = InventorySyncOptionsBuilder.of(CTP_TARGET_CLIENT)
                                                                                      .build();
         final InventorySync inventorySync = new InventorySync(inventorySyncOptions);
 
-
+        // benchmark
         final long beforeSync = System.currentTimeMillis();
         final InventorySyncStatistics inventorySyncStatistics =
             executeBlocking(inventorySync.sync(inventoryEntryDrafts));
         final long totalTime = System.currentTimeMillis() - beforeSync;
 
 
-        // Caclulate sync time and assert on threshold
-        final double diff = calculateDiff(SyncSolutionInfo.LIB_VERSION, INVENTORY_SYNC, CREATES_ONLY, totalTime);
-        assertThat(diff).withFailMessage(format("Diff of benchmark '%e' is longer than expected"
-                            + " threshold of '%d'.", diff, THRESHOLD))
-                        .isLessThanOrEqualTo(THRESHOLD);
+        // assert on threshold (based on history of benchmarks; highest was ~9 seconds)
+        final int threshold = 69000; // 1 minute higher than highest benchmark
```

Comment:
- Maybe instead of `1 minute` -> `~1 minute` in the comment?

> Also why would you use such high threshold if from history you know that it was never beyond 9s?

makes sense.  Will change.

Reply:
> Maybe instead of 1 minute -> ~1 minute in the comment

Why not the truth right away? :) -> "69 seconds"

Reply:
17c8f44

Reply:
Just to explain the choice of the magic number :)